languages/csharp/exercises/concept/arrays/.meta/Example.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+
+public class BirdCount
+{
+    private int[] birdsPerDay;
+
+    public BirdCount(int[] birdsPerDay)
+    {
+        this.birdsPerDay = birdsPerDay;
+    }
+
+    public int Total()
+    {
+        var total = 0;
+
+        foreach (var count in birdsPerDay)
+        {
+            total += count;
+        }
+
+        return total;
+    }
+
+    public int BusyDays()
+    {
+        var days = 0;
+
+        foreach (var count in birdsPerDay)
+        {
+            if (count >= 5)
+            {
+                days++;
+            }
+        }
+
+        return days;
+    }
+
+    public int Yesterday()
+    {
+        return birdsPerDay[5];
+    }
+
+    public bool HasDayWithoutBirds()
+    {
+        return Array.IndexOf(birdsPerDay, 0) != -1;
+    }
+
+    public static int[] LastWeek()
+    {
+        return new int[] { 0, 2, 5, 3, 7, 8, 4 };
+    }
+}
```

Comment:
Is it best to have a new-line at the end of all files? I didn't follow the full discussions there on whether it matters or not 😄. If yes then same applies to the other code files.
```suggestion
}

```

Reply:
It is. I'm working on integrating `dotnet format`, so we can do this easily next time :)